common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -52,13 +52,13 @@ class VideoFields(object):
         default=""
     )
     start_time = RelativeTime(  # datetime.timedelta object
-        help=_("Time you want the video to start if you don't want the entire video to play. Formatted as HH:MM:SS. The maximum value is 23:59:59."),
+        help=_("Time you want the video to start if you don't want the entire video to play. Not supported in the native mobile app: the full video file will play. Formatted as HH:MM:SS. The maximum value is 23:59:59."),
```

Comment:
Looks like these are too long: https://build.testeng.edx.org/job/edx-platform-all-tests-pr-flow/3819/Diff_Quality_Report/

Would recommend like this:
```python
help=_(
    "text text text "
    "yet mooooore text"
),
```